node/deno_config.ts

```diff
@@ -0,0 +1,52 @@
+import { promises as fs } from 'fs'
+import { join, resolve } from 'path'
+
+import { parse as parseJSONC } from 'jsonc-parser'
+
+import { isNodeError } from './utils/error.js'
+
+interface DenoConfigFile {
+  importMap?: string
+}
+
+const filenames = ['deno.json', 'deno.jsonc']
+
+export const getConfig = async (basePath?: string) => {
+  if (basePath === undefined) {
+    return
+  }
+
+  for (const filename of filenames) {
+    const candidatePath = join(basePath, filename)
+    const config = await getConfigFromFile(candidatePath)
+
+    if (config !== undefined) {
+      return normalizeConfig(config, basePath)
+    }
+  }
+}
+
+const getConfigFromFile = async (filePath: string) => {
+  try {
+    const data = await fs.readFile(filePath, 'utf8')
+    const config = parseJSONC(data) as DenoConfigFile
```

Comment:
should we do some minimal validation on the JSON?

Reply:
Done in https://github.com/netlify/edge-bundler/pull/256/commits/0c8d8477c620e242b5366513018fe5f3613b43e8.